src/EditorFeatures/CSharp/NavigationBar/CSharpNavigationBarItemService.cs

```diff
@@ -28,23 +26,6 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.NavigationBar
     [ExportLanguageService(typeof(INavigationBarItemService), LanguageNames.CSharp), Shared]
     internal class CSharpNavigationBarItemService : AbstractNavigationBarItemService
```

Comment:
i don't see the VisualBasicNavigationBarItemService change in this PR.

Reply:
Correct, there is not.